packages/ihe-gateway-sdk/src/client/ihe-gateway.ts

```diff
@@ -1,41 +1,128 @@
-import { Patient } from "@medplum/fhirtypes";
+import axios, { AxiosInstance } from "axios";
 import { v4 as uuidv4 } from "uuid";
-import { NPIStringArray, npiStringArraySchema } from "../models/shared";
-import { XCPDGateways, XCPDRequest, xcpdGatewaysSchema } from "../models/xcpd-request";
+import { npiStringArraySchema } from "../models/shared";
+import { XCPDPayload, XCPDRequest, xcpdGatewaysSchema, XCPDResponse } from "../models/xcpd";
+import { XCA_ITI_38Request } from "../models/xca_iti_38";
+import { XCA_ITI_39Request } from "../models/xca_iti_39";
+
+const DEFAULT_AXIOS_TIMEOUT_SECONDS = 120;
```

Comment:
Can we use dayjs' duration here, please? Then we can remove "seconds" from the name and convert the duration to whatever unit we want further down the code.